packages/medusa/src/utils/remote-query-fetch-data.ts

```diff
@@ -47,12 +86,12 @@ export function remoteQueryFetchData(container: MedusaContainer) {
 
     const result = await service[methodName](filters, options)
 
-    if (hasPagination) {
+    if (hasPagination_) {
       const [data, count] = result
       return {
         data: {
           rows: data,
-          metadata: {},
+          metadata: buildPagination(options, count),
```

Comment:
question: enables pagination and filters on `remoteQuery`?

Reply:
These are to correct identify pagination for internal services not loaded as modules.
To behave the same as it does for modules.